requirements.txt

```diff
@@ -4,3 +4,4 @@ jinja2>=2.9.6
 colorama>=0.3.7
 yapf==0.22.0
 pexpect>=4.2.1
+timeout-decorator>=0.4.0
```

Comment:
Do you really need this? If so, we need to package it (please open an issue), add it to dependencies.ini etc.

Reply:
maybe not... I needed it while I was writing the tests... so the problem here is that if this fails, that is if the test fails, it will just hang until you kill it...

that's why uI added this, the tests run smoothly now, so you'll know if you broke something when the test just doesn't complete....